style/pileup.css

```diff
@@ -119,7 +124,7 @@
 /* pileup track */
 .pileup-root > .pileup {
   flex: 1;  /* stretch to fill remaining space */
-  overflow-y: auto;
+  border-top: 1px solid #666;
```

Comment:
I agree that borders on every track looks a bit excessive. How does the margin approach look? I happily defer to your judgment on this one :)